src/cdk/testing/testbed/testbed-harness-environment.ts

```diff
@@ -54,7 +57,9 @@ export class TestbedHarnessEnvironment extends HarnessEnvironment<Element> {
   }
 
   private async _stabilize(): Promise<void> {
-    this._fixture.detectChanges();
-    await this._fixture.whenStable();
+    if (!this._destroyed) {
```

Comment:
Alternatively we could throw a better error message here, but I opted to just let it continue silently, because retrieving things from the DOM elements should still work, it's just that we can't run change detection anymore.

Reply:
I'm curious how you discovered this. Is it a mistake users would be likely to make? If so, maybe an error would be useful to save them some debugging time later

Reply:
I ran into it when I wanted to log something out in a `forEach` and I forgot to `await` a call to get the text of the element. It ended up being invoked after the fixture was destroyed.

Reply:
Interesting scenario. I feel like this could be a common issue if someone just forgets the `await`. In that case I'd also agree that an error will be more helpful (helping to raise awareness)